Tasks/GitHubReleaseV0/Tests/UtilityL0Tests.ts

```diff
@@ -0,0 +1,86 @@
+import { Utility } from "../operations/Utility";
+import { TestString } from "./TestStrings";
+
+export class UtilityL0Tests {
+    public static startTests() {
+        this.validateGetReleaseNote();
+        this.validateNormalizeBranchName();
+        this.validateParseHTTPHeaderLink();
+        this.validateExtractRepositoryOwnerAndName();
+        this.validateExtractRepoAndIssueId();
+        this.validateGetFirstLine();
+    }
+
+    public static validateGetReleaseNote() {
+        let releaseNoteInput = "release_note_input";
+        let changeLog = "change_log";
+
+        let releaseNote = Utility.getReleaseNote("input", null, releaseNoteInput, changeLog);
+        if (releaseNote === (releaseNoteInput + changeLog)) {
+            console.log(TestString.getReleaseNoteKeyword);
+        }
+    }
+
+    public static validateNormalizeBranchName() {
+        let normalizedBranchName = "";
+
+        normalizedBranchName = Utility.normalizeBranchName("refs/tags/tagName");
+        if (normalizedBranchName === "tagName") {
+            console.log(TestString.validBranchNameKeyword);
+        }
+
+        normalizedBranchName = Utility.normalizeBranchName("refs/heads/tagName");
+        if (!normalizedBranchName) {
+            console.log(TestString.invalidBranchNameKeyword);
+        }
+    }
+
+    public static validateParseHTTPHeaderLink() {
+        let parsedHttpHeaderLink = {};
+
+        let headerLink = '<https://api.github.com/search/code?q=addClass+user%3Amozilla&page=2>; rel="next", <https://api.github.com/search/code?q=addClass+user%3Amozilla&page=34>; rel="last"';
+        let expectedParsedHeaderLink = {
+            "next": "https://api.github.com/search/code?q=addClass+user%3Amozilla&page=2",
+            "last": "https://api.github.com/search/code?q=addClass+user%3Amozilla&page=34"
+        }
+        parsedHttpHeaderLink = Utility.parseHTTPHeaderLink(headerLink);
+
+        if (JSON.stringify(parsedHttpHeaderLink) === JSON.stringify(expectedParsedHeaderLink)) {
+            console.log(TestString.parseHTTPHeaderLinkKeyword);
+        }
+    }
+
+    public static validateExtractRepositoryOwnerAndName() {
+        let repoName = "owner_name/repo_name";
+
```

Comment:
remove necessary lines. Applicable at multiple places.

Reply:
Will take in next PR